src/clj/rems/css/styles.clj

```diff
@@ -27,6 +27,14 @@
 (def logo-height-menu (u/px 40))
 (def logo-height (u/px 150))
 (def menu-height 56)
+(def media-breakpoints {:small-screen (u/px 480)
+                        :medium-screen 870
+                        :big-screen 1080})
```

Comment:
This seems unused?

Reply:
Also one is in pixels, one in plain numbers?

Reply:
I wanted to store the current media breakpoints so that we still have some track of them untill I am sure new breakpoints work well.